Modules/_randommodule.c

```diff
@@ -261,6 +261,8 @@ random_seed_time_pid(RandomObject *self)
 
 #ifdef HAVE_GETPID
     key[2] = (uint32_t)getpid();
+#elif defined(MS_WINDOWS_UWP)
+    key[2] = (uint32_t)GetCurrentProcessId();
```

Comment:
This case will need to be ordered above `HAVE_GETPID` with the changes mentioned in other comments. There's no problem with that.